### DISCUSSION TITLE
Minimum Stability

### DISCUSSION THREAD

**Discussion by timothygwebb** [boxbilling/boxbilling#1205] [General]
I found this on slack and it goes along with my pull request comments from the other day. Let's discuss and come to a resolution to where we can always state the minimum stability of our product may be found in composer.json. By default minimum stability should be stable but as questioned below it does not seem to be. May we clearly define this as stable or beta in the minimum stability.

https://boxbilling.slack.com/files/U03Q5VB8R/F02S9JGSXGE/image_from_ios.png

https://github.com/boxbilling/boxbilling/pull/1201

**Comment by yagiz-dev** [comment]
Locking this as we already have issues and pull requests tracking this.
